docs/blog/articles/2023-02-05-KServe-0.10-release.md

```diff
@@ -0,0 +1,114 @@
+# Announcing: KServe v0.10.0
+
+We are excited to announce KServe 0.10 release, in this release we have been focusing on enabling more KServe networking setup options,
+improving metrics for supported serving runtimes and increasing support coverage for [Open(aka v2) inference protocol](https://kserve.github.io/website/0.10/modelserving/data_plane/v2_protocol/) for both KServe and ModelMesh.
```

Comment:
model mesh is also kserve component right ? so maybe different wording here - like for single, multi model and modelmesh?

Reply:
reworded to standard and modelmesh InferenceService